### ISSUE TITLE
Ubuntu apt-get gives 404 error for repo URL

### ISSUE THREAD

**Issue by akwala** [vector-im/element-web#18349]
#### Description
`sudo apt-get upgrade element-desktop` gives

```
Err:1 https://packages.riot.im/debian default/main amd64 element-desktop amd64 v1.7.35
  404  Not Found [IP: 104.26.5.138 443]
E: Failed to fetch https://packages.riot.im/debian/pool/main/e/element-desktop/element-desktop_v1.7.35_amd64.deb  404  Not Found [IP: 104.26.5.138 443]
```
Probable duplicate of or related to #18326 
#### Steps to reproduce

Run `sudo apt-get upgrade element-desktop`

/etc/apt/sources.list.d/riot-im.list:
```
deb [signed-by=/usr/share/keyrings/riot-im-archive-keyring.gpg] https://packages.riot.im/debian/ default main
```
Describe how what happens differs from what you expected.
See Description.

#### Version information

- **Platform**: desktop

For the web app:

NA

For the desktop app:

- Kubuntu 20.04
- Element version: 1.7.33; olm version: 3.2.3

**Comment by lorrin** [comment]
One thing that jumps out at me is that APT is trying to fetch 1.7.35 but the just-released version is 1.7.34.

**Comment by robintown** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/18345
